src/kaltura-player.js

```diff
@@ -280,12 +316,14 @@ class KalturaPlayer extends FakeEventTarget {
   destroy(): void {
     const targetId = this.config.ui.targetId;
     this._reset = true;
+    this._playbackStart = false;
```

Comment:
I think you need to change it on `reset` as well

Reply:
it is